ruby-training/trainees/alan/test_katas/03/JadenCase/jaden_case.rb

```diff
@@ -0,0 +1,16 @@
+
+
+class JadenCase
+    
+    def self.to_array_of_words (sentence)
```

Comment:
I think this method is unnecessary, you only call a single method the language already has with the same purpose (split()). You're essentially adding an extra layer of methods that doesn't "add" any logic.